packages/smooth_app/lib/pages/preferences/user_preferences_languages_list.dart

```diff
@@ -0,0 +1,417 @@
+// Flutter Supported Languages List
+// https://api.flutter.dev/flutter/dart-ui/Locale-class.html
+
+List<String> languageNameList = <String>[
```

Comment:
The list of supported languages is probably `OpenFoodFactsLanguage.values`.
I have no idea how you can hope that the order of this `List` will make it reusable.
And it's not a list of languages here, it's a list of English translations.
A `Map` would make more sense.

Reply:
OpenFoodFactsLanguage.values return List of OpenFoodFactsLanguage i.e [OpenFoodFactsLanguage.ENGLISH ...]
I don't know how to search for languages English names with that.
If I had used a Map instead of a List the filter method would take O(map size) operation for each comparison.
```dart
map.forEach((key, value) {
    if (key.contains('a')) {
      print('$key: $value');
    };
  });
```
Or get all the keys and store them as List<String>.
Can you suggest me a better way?